src/coord/tests/sql.rs

```diff
@@ -43,14 +47,26 @@ async fn datadriven() {
                 "add-table" => {
                     let id = catalog.allocate_user_id().unwrap();
                     let oid = catalog.allocate_oid().unwrap();
+                    let database_id = catalog
+                        .resolve_database(DEFAULT_DATABASE_NAME)
+                        .unwrap()
+                        .id();
+                    let database_spec = ResolvedDatabaseSpecifier::Id(database_id);
+                    let schema_spec = catalog
+                        .resolve_schema_in_database(&database_spec, DEFAULT_SCHEMA, 0)
```

Comment:
This `0` probably wants to be `SYSTEM_CONN_ID` or somesuch.

Reply:
Yep, switched.